package.json

```diff
@@ -50,6 +50,7 @@
   "dependencies": {
     "@fontsource/open-sans": "^4.5.1",
     "babel-plugin-react-require": "^3.1.3",
+    "moment": "^2.29.1",
```

Comment:
In the project there is already date-fns as a dependency for time manage, it is a tool that solves many of the problems that momentjs has. Even momentjs recommends its use. https://momentjs.com/docs/#/-project-status/